src/wp-admin/includes/class-wp-site-health-auto-updates.php

```diff
@@ -66,7 +66,7 @@ static function ( $test ) {
 	 * @param string $constant         The name of the constant to check.
 	 * @param bool|string|array $value The value that the constant should be, if set,
 	 *                                 or an array of acceptable values.
-	 * @return array The test results.
+	 * @return ?array The test results. Null if the test passed.
```

Comment:
It feels wrong to read "Null" even though I understand that you did this to follow WPCS.

How about this instead?

```suggestion
	 * @return ?array The test results. Or null if the test passed.
```

Similar concerns for all other occurances of 'Null'

Reply:
Hey @apermo thanks for the feedback but not fully sure I understood it 😅

Is the issue that the `N` is capitalized, or that it's not a proper sentence? 

If it's the latter, then probably better if I switch out `Or` with `Returns`, or the `.` with a comma.

I know it's low-stakes, but would be helpful to understand the conventions here for my future contribs.

Reply:
My concern is, that `null` is a technical term, and that's why I'd prefer a sentence where we can have `null` inside rather than `Null`. 

Adding the `Or` was just an idea, a `,` or any other construction is just as fine.

Reply:
Resolved in 5513d245e7615373a7044af19df3b7100136ee56

Reply:
Now you added an inconsistency. `, or ` vs `or `. I don't know which better, but imho we should stick to one.

Reply:
🤦added the missing commas in https://github.com/WordPress/wordpress-develop/pull/7675/commits/4a6e02f6f311504aef102f0412ad2a3d2be459a3